packages/zowe-explorer-api/package.json

```diff
@@ -10,8 +10,7 @@
   },
   "main": "lib/index.js",
   "files": [
-    "lib",
-    "src"
+    "lib"
```

Comment:
@zFernand0 does changing this is keeping source out of the packaging for npmjs?

Reply:
That's correct 😋